src/Equinox.Projection.Kafka/Kafka.fs

```diff
@@ -317,6 +330,22 @@ type KafkaConsumer private (log : ILogger, consumer : Consumer<string, string>,
             ConsumerBuilder<_,_>(config.Kvps)
                 .SetLogHandler(fun _c m -> log.Information("consumer_info|{message} level={level} name={name} facility={facility}", m.Message, m.Level, m.Name, m.Facility))
                 .SetErrorHandler(fun _c e -> log.Error("Consuming... Error reason={reason} code={code} broker={isBrokerError}", e.Reason, e.Code, e.IsBrokerError))
+                .SetStatisticsHandler(fun _c json -> 
+                    // Stats format: https://github.com/edenhill/librdkafka/blob/master/STATISTICS.md
+                    let stats = JToken.Parse json
+                    (stats.Item "topics").Children()
+                    |> Seq.choose(fun t -> 
```

Comment:
combine choose and filters (in fact probably do a `seq { for .. if -> ` comprehension)